charts/nats/templates/nats-jetstream-tls-secret.yaml

```diff
@@ -0,0 +1,40 @@
+######################################
+##  Nats Jetstream SSL Certificate  ##
+######################################
+{{ if and  .Values.global.nats.jetStream.enabled .Values.global.nats.jetStream.tls }}
+{{- $ca := genCA "ca" 3650 }}
```

Comment:
Oh dude, this `genCA` function led me to this, which I think can vastly simplify all of this certificate stuff: https://helm.sh/docs/chart_template_guide/function_list/#cryptographic-and-security-functions

Reply:
reworking the changes - i was previously using this function but wasnt using right config to make services up - will alter the changes reduce the code in here

Reply:
reworked the changes and removed the custom script

Reply:
@danielhoherd this job template should in common template so that i can use the hash to restart houston and houston-worker if it requires a restart due to this change - currently worker may not get restart if there is change in certs